src/Runner.ts

```diff
@@ -43,7 +48,8 @@ export class Runner {
     }
     while (
```

Comment:
let's set aborted to false when the run is invoked?

Reply:
Makes sense! though I'm not sure how we're expecting people to use this `abort` functionality since after `abort` we might say that you can't run the same runner anymore.

The reason I'm thinking this is, if a run is completed successfully with `await runner.run()` the consecutive `run` call won't do anything since `this.#nextStep` will point to the end of the flow.

So, maybe we can rename `abort` to `stop` and when they `run` after calling `abort` it continues from where it is left. WDYT?

Reply:
Removed `stepIdx` from `run` and always start from the beginning for `run`